package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "root",
-  "private": true,
+  "name": "@gravitywelluk/packages",
+  "private": false,
```

Comment:
should still be private true as its not a published package i believe? not sure it makes much difference but lerna uses the field